assets/js/components/Project/List.jsx

```diff
@@ -49,9 +52,15 @@ const ProjectList = () => {
     setModalOpen(true);
   };
 
+  const onEditProject = (e, project) => {
+    setActiveProject(project);
+    setShowForm(!showForm);
+  };
+
   const onCloseModal = () => {
     setActiveModal(null);
     setModalOpen(false);
+    setActiveProject();
```

Comment:
Let's feed `setActiveProject()` a value.  Either `null` or an empty string `''` if it's a string data type.